cypress/e2e/GUI/gui.cy.js

```diff
@@ -0,0 +1,230 @@
+        cy.contains('Hotmart').should('be.visible')
+        cy.contains('Udemy').should('be.visible')
+        cy.contains('Blog').should('be.visible')
+        cy.contains('YouTube').should('be.visible')
+    })
+
+    it('Verifica se e possivel alterar entre os temas corretamente', () => {
+
+        //Clica no bot�o de trocar o tema e checa se trocou corretamente para Dark
+        cy.get('.ThemeToggle_container__RIfhe').click()
+        cy.get('body[data-theme="dark"]').should('be.visible')
+
+        //Clica no bot�o de trocar o tema e checa se voltou corretamente para Light
+        cy.get('.ThemeToggle_container__RIfhe').click()
+        cy.get('body[data-theme="light"]').should('be.visible')
+    })
+
+    it('Verifica se o tema se mantem ao acessar a informacao de cliente corretamente', () => {
+
+        //Clica no bot�o de trocar o tema e checa se trocou corretamente para Dark
+        cy.get('.ThemeToggle_container__RIfhe').click()
+        cy.get('body[data-theme="dark"]').should('be.visible')
+
+        //Clica para acessar os detalhes de um cliente e verifica se foi realizado corretamente
+        cy.contains('button', 'View').click()
+        cy.contains('Customer Details').should('be.visible')
+        cy.get('body[data-theme="dark"]').should('be.visible')
+    })
+
+    it('Mantem os filtros ao voltar da visualizacao de detalhes do cliente', () => {
+
+        //Realiza os filtros corretamente
+        cy.get('select[data-testid="size-filter"]').select('Small')
+        cy.get('select[data-testid="industry-filter"]').select('Logistics')
+
+        //Acessa um cliente apresentado e checa se acessou
+        cy.contains('button', 'View').click()
+        cy.contains('Customer Details').should('be.visible')
+
+        //Retorna e confirma que retornou para a p�gina anterior
+        cy.contains('button', 'Back').click({ force: true })
+        cy.contains('Below is our customer list.').should('be.visible')
+
+        //Verifica se o filtro segue os mesmos selecionados anteriormente
+        cy.get('select[data-testid="size-filter"]').should('have.value', 'Small')
+        cy.get('select[data-testid="industry-filter"]').should('have.value', 'Logistics')
+    })
+
+    it('Verifica se e possivel acessar os detalhes do cliente', () => {
+
+        //Clica para acessar os detalhes de um cliente e verifica se foi realizado corretamente
+        cy.contains('button', 'View').click()
+        cy.contains('Customer Details').should('be.visible')
+    })
+
+    it('Verifica se e possivel expandir e recolher o endereco do cliente', () => {
+
+        //Clica para acessar os detalhes de um cliente e verifica se foi realizado corretamente
+        cy.contains('button', 'View').click()
+        cy.contains('Customer Details').should('be.visible')
+
+        //Clica no bot�o "Show Address" para expandir o endere�o do cliente e verifica se expandiu
+        cy.contains('Show address').click({ force: true })
```

Comment:
Por qual motivo você está forçando o clique. Isso é uma má prática a qual deve ser evitada em 99% por dos casos, e este não é uma excessão.